workflows/prognostic_run_diags/fv3net/diagnostics/prognostic_run/views/matplotlib.py

```diff
@@ -89,7 +101,73 @@ def plot_2d_matplotlib(
         template.render(
             image_data=data,
             runs=sorted(run_diags.runs),
-            variables_to_plot=variables_to_plot,
+            variables_to_plot=sorted(variables_to_plot),
+            varfilter=varfilter,
+        )
+    )
+
+
+def plot_cubed_sphere_map(
+    run_diags: RunDiagnostics,
+    run_metrics: RunMetrics,
+    varfilter: str,
+    metrics_for_title: Mapping[str, str] = None,
+    **opts,
+) -> str:
+    """Plot horizontal maps of cubed-sphere data for diagnostics which match varfilter.
+    
+    Args:
+        run_diags: the run diagnostics
+        run_metrics: the run metrics, which can be used to annotate plots
+        varfilter: pattern to filter variable names
+        metrics_for_title: metrics to put in plot title. Mapping from label to use in
+            plot title to metric_type.
+        opts: additional kwargs to pass to fv3viz.plot_cube
+
+    Note:
+        All matching diagnostics must have tile, x and y dimensions and each dataset in
+        run_diags must include lat/lon/latb/lonb coordinates.
+    """
+
+    data = defaultdict(dict)
+    if metrics_for_title is None:
+        metrics_for_title = {}
+
+    variables_to_plot = run_diags.matching_variables(varfilter)
+
+    for run in run_diags.runs:
+        for varname in variables_to_plot:
+            logging.info(f"plotting {varname} in {run}")
+            shortname = varname.split("_")[0]
+            ds = run_diags.get_variables(run, list(_COORD_VARS) + [varname])
+            plot_title = _render_map_title(
+                run_metrics, shortname, run, metrics_for_title
+            )
+            fig, ax = plt.subplots(
+                figsize=(6, 3), subplot_kw={"projection": ccrs.Robinson()}
+            )
+            mv = fv3viz.mappable_var(ds, varname, coord_vars=_COORD_VARS, **COORD_NAMES)
+            fv3viz.plot_cube(mv, ax=ax, **opts)
+            ax.set_title(plot_title)
+            plt.subplots_adjust(left=0.01, right=0.75, bottom=0.02)
+            data[varname][run] = fig_to_b64(fig)
+            plt.close(fig)
+    return raw_html(
+        template.render(
+            image_data=data,
+            runs=sorted(run_diags.runs),
+            variables_to_plot=sorted(variables_to_plot),
             varfilter=varfilter,
         )
     )
+
+
+def _render_map_title(
+    metrics: RunMetrics, variable: str, run: str, metrics_for_title: Mapping[str, str],
+) -> str:
+    title_parts = []
+    for name_in_figure_title, metric_type in metrics_for_title.items():
+        metric_value = metrics.get_metric_value(metric_type, variable, run)
+        metric_units = metrics.get_metric_units(metric_type, variable, run)
+        title_parts.append(f"{name_in_figure_title}: {metric_value:.3f}{metric_units}")
```

Comment:
adds a space before the units:
```suggestion
        title_parts.append(f"{name_in_figure_title}: {metric_value:.3f} {metric_units}")
```